tests/config/raw_overlap_partition_table.yaml

```diff
@@ -0,0 +1,18 @@
+api-version: 0
+disklabel: gpt
+
+raw:
+  - input-offset: 0
+    output-offset: 10KiB
```

Comment:
How about reducing the output offset to zero? This way, it's more obvious that it is actually overlapping the partition table, regardless of the partition table type.

Reply:
Done.